### ISSUE TITLE
PostgreSQL composite primary key generate primary key attribute only on one column

### ISSUE THREAD

**Issue by Hinidu** [linq2db/linq2db#139]
After model generation with ```linq2db/t4models``` only the first column of a composite primary key has the PrimaryKey attribute. It can be fixed manually by ```GetColumn(tableName, column2Name).IsPrimaryKey = true;```, so it seems like a bug in ```PostgreSQLSchemaProvider.GetPrimaryKeys``` rather than a bug in t4 templates. Suddenly I don't have enough experience with PostgreSQL to fix it myself quickly.
Thank you in advance.

**Comment by Ysilla** [comment]
Having the same issue with PostgreSQL. Also not sure if related to this issue but it also fails completely at associations mapping. With the following example, a basic one to many relation, 1 map to multiple segments (with segments having a PK on 2 columns). the standard generated code will not only have the PK on just map_id, but will also have the whole relation in the wrong direction, map object has one segment attribute, and segment object has an IEnumerable<map>, should be the opposite.

```SQL
CREATE TABLE maps
(
  map_id integer NOT NULL,
  CONSTRAINT maps_pkey PRIMARY KEY (map_id)
)

CREATE TABLE segments
(
  map_id integer NOT NULL,
  segment_index integer NOT NULL,
  CONSTRAINT segments_pkey PRIMARY KEY (map_id, segment_index),

  CONSTRAINT segment_map_id_fkey
    FOREIGN KEY (map_id)
      REFERENCES maps (map_id) MATCH SIMPLE
      
  ON UPDATE NO ACTION ON DELETE NO ACTION
)
```

Couldn't find any way for this to work properly, even after fixing the primary key issue, the attributes created from the FK always remain null on both elements.

**Comment by ckarumoreira** [comment]
Same thing here, only the first column is marked as primary key. Has someone found out a way to solve it (even outside of the t4models)?

**Comment by Hinidu** [comment]
@ckarumoreira I provided the workaround in the description of the issue:
It can be fixed manually by `GetColumn(tableName, column2Name).IsPrimaryKey = true;` in t4 template.

**Comment by igor-tkachev** [comment]
Fixed with #215.
